### PR TITLE
[Fluent] Scrollview

### PR DESCRIPTION
**Pull request by magdalenaan**
related https://github.com/telerik/kendo-themes-private/issues/3

### CODE REVIEW

File: packages/fluent/scss/scrollview/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+$kendo-scrollview-navigation-color: $white !default;
+/// Scrollview navigation arrows shadow.
+/// @group scrollview
+$kendo-scrollview-navigation-arrow-shadow: var( --kendo-box-shadow-depth-3, none ) !default;
+/// Scrollview navigation background color.
+/// @group scrollview
+$kendo-scrollview-navigation-bg: transparent !default;
+/// Scrollview navigation hover span background color.
+/// @group scrollview
+$kendo-scrollview-navigation-hover-span-bg: null !default;
```
Comment (Juveniel) [inline packages/fluent/scss/scrollview/_variables.scss:99]: set a default value

File: packages/fluent/scss/scrollview/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+/// Font size of the scrollview.
+/// @group scrollview
+$kendo-scrollview-font-size: var( --kendo-font-size, inherit ) !default;
+/// Line height of the scrollview.
+/// @group scrollview
+$kendo-scrollview-line-height: var( --kendo-line-height, normal ) !default;
+
+/// Background color of the scrollview.
+/// @group scrollview
+$kendo-scrollview-bg: transparent !default;
```
Comment (Juveniel) [inline packages/fluent/scss/scrollview/_variables.scss:19]: can we use --component-bg here ? Is there any specific reason why you have set transparent ?

File: packages/fluent/scss/scrollview/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+/// Scrollview pager offset.
+/// @group scrollview
+$kendo-scrollview-pager-offset: 0 !default;
+/// Scrollview pager item spacing.
+/// @group scrollview
+$kendo-scrollview-pager-item-spacing: 20px !default;
+/// Scrollview pager item border width.
+/// @group scrollview
+$kendo-scrollview-pager-item-border-width: 0 !default;
```
Comment (Juveniel) [inline packages/fluent/scss/scrollview/_variables.scss:63]: 0px

File: packages/fluent/scss/scrollview/_variables.scss (hunk 1 above)
Comment (kikostadinov) [reply packages/fluent/scss/scrollview/_variables.scss:99]: Looks like this variable is no longer used, so I had it removed.

File: packages/fluent/scss/scrollview/_variables.scss (hunk 2 above)
Comment (kikostadinov) [reply packages/fluent/scss/scrollview/_variables.scss:19]: There is no specific reason.

**Review by Juveniel — APPROVED** [review]

**Pull request merged**